geode-core/src/test/java/org/apache/geode/management/internal/cli/shell/GfshExecutionStrategyTest.java

```diff
@@ -102,6 +112,11 @@ public Result offlineCommand() {
       return ResultBuilder.createInfoResult(COMMAND1_SUCCESS);
     }
 
+    @CliMetaData(shellOnly = true)
+    public ResultModel offlineCommand2() {
+      return ResultModel.createInfo(COMMAND3_SUCCESS);
```

Comment:
The naming of these dummy commands and the corresponding results is confusing. For instance, I think renaming `COMMAND3_SUCCESS` to `OFFLINE_COMMAND2_SUCCESS` is more meaningful. SImilar for the other two dummy commands.